dali/operators/reader/loader/cufile_loader.h

```diff
@@ -40,7 +40,7 @@ class CUFileLoader : public FileLoader<GPUBackend, Target, CUFileStream> {
  public:
   explicit CUFileLoader(const OpSpec& spec, vector<std::string> images = {},
                         bool shuffle_after_epoch = false)
-      : FileLoader<GPUBackend, Target, CUFileStream>(spec) {
+      : FileLoader<GPUBackend, Target, CUFileStream>(spec, shuffle_after_epoch) {
```

Comment:
Btw, if every usage of FileLoader requires this field to be propagated, can we remove the default from constructor to prohibit the error from happening again?

Reply:
Done